cassiopeia/simulator/SequentialLineageTracingDataSimulator.py

```diff
@@ -0,0 +1,280 @@
+"""
+A Sequential lineage tracing data simulator. This is a subclass of the
+LineageTracingDataSimulator that simulates the data produced from sequential
+Cas9-based technologies (e.g, Typewriter described in Choi et al, Nature 2022). 
+This simulator implements the method `overlay_data` which takes
+in a CassiopeiaTree with edge lengths and overlays states.
+"""
+import copy
```

Comment:
a couple of imports don't seem to be used here -- `copy`, `Callable`, `Tuple`, and `Union`. Let's remove them if they're not used.